src/Shared.Core/KoikatuAPIBase.cs

```diff
@@ -86,8 +85,7 @@ private void BaseAwake()
             Logger = base.Logger;
 
             EnableDebugLoggingSetting = Config.Bind("Debug", "Show debug messages", false, "Enables display of additional log messages when certain events are triggered within KKAPI. Useful for plugin devs to understand when controller messages are fired. Changes take effect after game restart.");
-            RememberFilePickerFolder = Config.Bind("General", "Remember last folder in file pickers", true, "If true, file picker dialogs will remember the last opened folder and open already inside of it. If false, the dialogs will always open in the default folder.\n\nWarning: This setting only applies to plugins that use the file picker through this API.");
-            RememberFilePickerSaveLoad = Config.Bind("General", "Save remembered file picker folder between restarts", true, "Save and load last remembered file picker folders across game restarts");
+            RememberFilePickerFolder = Config.Bind("General", "Remember last folder in file pickers", RememberFilePickerFolderSetting.Enabled, "If enabled, file picker dialogs will remember the last opened folder and open already inside of it. If disabled, the dialogs will always open in the default folder.\n\nReset On Restart is enabled, but resets the remembered paths between restarts\n\nWarning: This setting only applies to plugins that use the file picker through this API.");
```

Comment:
```suggestion
            RememberFilePickerFolder = Config.Bind("General", "Remember last folder in file pickers", RememberFilePickerFolderSetting.Enabled, "If enabled, file picker dialogs will remember the last opened folder and open inside of it instead of the default folder. If disabled, the dialogs will always open in the default folder.\n\n'Reset On Restart' will remember the last opened folders only until the game is restarted, after which they will be reset.\n\nWarning: This setting only applies to plugins that use the file picker through this API.");
```